cmd/export.go

```diff
@@ -41,6 +41,8 @@ var exportCmd = &cobra.Command{
 }
 
 var config *whisk.Config
+var wskpropsPath string
```

Comment:
these two variables were defined in `cmd/report.go` which are not needed for report command but used by export and therefore adding them to `cmd/export.go`